Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 ruby "2.2.2"
 
 gem "rails", "~> 5.0.0"
+gem "active_model_serializers"
```

Comment:
Since there are big API changes between versions, I think this should be pessimistically locked.